tests/fixtures/model_fixture.py

```diff
@@ -59,10 +60,52 @@ def prepare_model_for_local_bucketfs(model_specification: ModelSpecification,
     download_model_to_path(current_model_specs, bucketfs_path_for_model)
     return tmpdir
 
+@pytest.fixture(scope="session")
+def prepare_filling_mask_model_for_local_bucketfs(tmpdir_factory) -> PurePosixPath:
+    model_specification = model_params.base_model_specs
+    model_specification.task_type = "fill-mask"
+    bucketfs_path = prepare_model_for_local_bucketfs(model_specification, tmpdir_factory)
+    yield bucketfs_path
 
 @pytest.fixture(scope="session")
-def prepare_base_model_for_local_bucketfs(tmpdir_factory) -> PurePosixPath:
+def prepare_question_answering_model_for_local_bucketfs(tmpdir_factory) -> PurePosixPath:
     model_specification = model_params.base_model_specs
```

Comment:
I think, you should select one model per task for testing, because the base_model "bert_uncased" can't be used for all tasks. I think, it can be used for mask filling, sequence classification, zero shot classification and maybe text generation. I think, it can't used for  translation, question answering.